hypothesis-python/tests/quality/test_zig_zagging.py

```diff
@@ -92,7 +89,7 @@ def test_function(data):
         settings=settings(base_settings, phases=(Phase.generate, Phase.shrink)),
     )
 
-    runner.cached_test_function(b + hbytes([0]) + b + hbytes([1]) + marker)
+    runner.cached_test_function(b + bytes([0]) + b + bytes([1]) + marker)
```

Comment:
We could replace all of our `hbytes([0])`  and similar with the corresponding literals e.g. `b'\0'`.

Reply:
I did look at these... we don't have a clear standard so far.  If you want to convert I'd probably standardise on `b"\x00\x01"` hex literal strings, rather than the `b"\0\1"` escapes.

Specifically though, I'd add that to the list in #2218 and do it after merging this :wink: